cypress/e2e/12-canvas-actions.cy.ts

```diff
@@ -198,4 +198,32 @@ describe('Canvas Actions', () => {
 		cy.get('body').type('{shift}', { release: false }).type('{leftArrow}');
 		WorkflowPage.getters.selectedNodes().should('have.length', 2);
 	});
+
+	it('should not break lasso selection when dragging node action buttons', () => {
+		WorkflowPage.actions.addNodeToCanvas(MANUAL_TRIGGER_NODE_NAME);
+		WorkflowPage.getters
+			.canvasNodes()
+			.last()
+			.find('[data-test-id="disable-node-button"]').as('disableNodeButton');
+		cy.get('@disableNodeButton').trigger('mousedown', { force: true });
+		cy.get('@disableNodeButton').trigger('mousemove', 200, 200, { force: true });
+		cy.get('@disableNodeButton').trigger('mouseup', { force: true });
+		WorkflowPage.actions.testLassoSelection([100, 100], [200, 200]);
+	});
+
+	it('should not break lasso selection with multiple clicks on node action buttons', () => {
+		WorkflowPage.actions.addNodeToCanvas(MANUAL_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.testLassoSelection([100, 100], [200, 200]);
+		WorkflowPage.getters
+			.canvasNodes()
+			.last().as('lastNode');
+		cy.get('@lastNode').find('[data-test-id="disable-node-button"]').as('disableNodeButton');
```

Comment:
Here and above you could use the Cypress method we aleady have: `findChildByTestId`